apache/apache.go

```diff
@@ -94,21 +76,33 @@ func getMetrics(webserver string, metrics []string) ([]plugin.MetricType, error)
 			if err != nil {
```

Comment:
Hmmm interesting! 
```
vagrant@vagrant-ubuntu-trusty-64:~$ curl http://localhost:80/server-status?auto 
Total Accesses: 1
Total kBytes: 0
CPULoad: .0293856
Uptime: 1123
ReqPerSec: .000890472
BytesPerSec: 0
BytesPerReq: 0
BusyWorkers: 1
IdleWorkers: 49
ConnsTotal: 0
ConnsAsyncWriting: 0
ConnsAsyncKeepAlive: 0
ConnsAsyncClosing: 0
Scoreboard: _______________W__________________________________....................................................................................................
```
My output is as shown above but I'm using apache 2.4.7. I spun up a container and added a test case for the above output. A more intelligent parsing solution is required, I'm totally on board with more intelligent metrics coming out of apache. I'll try and work with parse metrics so it can take any version of an apache server and make it work. While at the same time ensuring we get proper units for the metrics that are currently advertised.